app/controllers/podcast_ownerships_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PodcastOwnershipsController < ApplicationController
```

Comment:
This controller doesn't have neither tests nor templates, I wonder if it should be added in its own separate PR, after we merge the table (it seems also off topic, as mentioned by the title)

Reply:
Hmm yeah.. Chris suggested that I add a Pundit policy and a controller too...hence I added it here...we can have it in a separate PR if required too!

Reply:
@rhymes I'll also be working on a follow up issue https://github.com/MLH-Fellowship/dev.to/issues/85
So I can remove the controller and the associated pundit policy from this PR if you wish

Reply:
Yeah, I think it'd be simpler to merge one bit at the time, this way we don't merge broken code (if anyone accesses the controller URL they will get a 400 error because the templates aren't there)